tag/map_test.go

```diff
@@ -184,20 +189,18 @@ func TestNewMapValidation(t *testing.T) {
 
 	for i, tt := range tests {
 		ctx := NewContext(context.Background(), tt.seed)
-		m, err := NewMap(ctx)
+		ctx, err := New(ctx)
+		m := FromContext(ctx)
```

Comment:
Let's move this line after line 203 when we know there isn't an error.

Reply:
Done.